### DISCUSSION TITLE
did you rewrite rofi UI?

### DISCUSSION THREAD

**Discussion by foodornt** [fdw/rofimoji#171] [Q&A]
To the creators of this project: I realize that you can run rofimoji both independently or using rofi,
did you rewrite rofi UI in python in order to do that? thanks in advance

**Comment by foodornt** [comment]
Despite my poor technical skills, I was able to follow along the code and understand what's going on, thanks to the elegant writing of it
https://github.com/fdw/rofimoji/blob/main/src/picker/selector/selector.py#L19C1-L20C1
Answer: no, rofi is not rewritten, it's launched from python directly
Thanks for the amazing project

**Reply by fdw** [reply]
You're welcome, and thanks for the praise 😊
